configs/components/medusa/medusa.yaml

```diff
@@ -0,0 +1,70 @@
+# MEDUSA-2.0 YAML CONFIGURATION FILE
+#
+model: medusa
+branch: main
+version: '2.0'
+apps: 'medusa_recom_awiesm'
+
+metadata:
+        Institute: University Liege
+        Description:
+                MEDUSA is Model of Early Diagenesis in the Upper Sediment with Adatable complexity.
+        Authors: Guy Munhoven
+        Publications:
+                - "Model of Early Diagenesis in the Upper Sediment with Adaptable complexity – MEDUSA (v. 2): a time-dependent biogeochemical sediment module for Earth system models, process analysis
+and teaching <https://doi.org/10.5194/gmd-14-3603-2021>"
+        License:
+                Please make sure you have a licence to use MEDUSA. In case you are unsure,
+                please contact Guy Munhoven (guy.munhoven@uliege.be)
+
+choose_version: 
+    '2.0':
+      branch: main
+
+git-repository: https://gitlab.dkrz.de/modular_esm/medusa2.git 
+# directory for code and libraries
+model_dir: "${general.esm_master.dir}/medusa-${version}" 
+#lib_dir: "${model_dir}/lib"
+#inc_dir: "${model_dir}/include"
+#code_dir: "${model_dir}/medmbm/trunk4mcg"
```

Comment:
Remove these three lines, as they are not used. In general that is also my only review suggestion: remove things that are not being used. Otherwise it might lead to confusion and people might ask "should we include this?".